man/bemenu.1.in

```diff
@@ -0,0 +1,311 @@
+In the following examples,
+.Cm C-x
+means
+.Aq Cm control-x ,
+.Cm M-x
+means
+.Aq Cm alt-x
+and
+.Cm S-x
+means
+.Aq Cm shift-x :
+.Bl -tag -width Ds
+.It Eo
+.Aq Cm cursor-up ,
+.Cm S- Ns Aq Cm cursor-left ,
+.Cm C-p , M-j , M-l Ec
+Select previous item.
+.It Eo
+.Aq Cm cursor-down ,
+.Aq Cm tab ,
+.Cm C-n , M-h , M-k Ec
+Select next item.
+.It Cm PageUp , M-v , M-u
+Show the previous page of items, preserving the offset of the currently
+highlighed item, if possible.
+.It Cm PageDown , C-v , M-d
+Show the next page of items, preserving the offset of the currently
+highlighed item, if possible.
+.It Cm S-M-< , S-PageUp
+Select first item in list.
+.It Cm S-M-> , S-PageDown
+Select last item in list.
+.It Cm S- Ns Aq Cm tab
+Copy the highlighed item to the filter.
+.It Eo
+.Aq Cm escape ,
+.Cm C-g Ec
+Terminate
+.Nm
+without printing items.
+.It Cm C- Ns Aq Cm carriage-return
+Select the highlighted item and continue selecting.
+.It Eo
+.Cm S- Ns Aq Cm carriage-return ,
+.Cm C-m Ec
+Print the selected and currently highlighed items, and exit.
+Items are printed in the order they were selected,
+with the currently highlighted item printed last.
+.It Eo
+.Cm S- Ns Aq Cm carriage-return ,
+.Cm Insert Ec
+Print the filter text to standard output and exit.
+.It Cm C-l , Aq Cm cursor-left
+Move cursor left.
+.It Cm C-f , Aq Cm cursor-right
+Move cursor right.
+.It Cm C-a , Home
+Move cursor to the start.
+.It Cm C-e , End
+Move cursor to the end.
+.It Eo
+.Aq Cm backspace
+.Cm C-h Ec
+Remove character before the cursor.
+.It Aq Cm delete
+Remove character at the cursor.
+.It Cm C-u , S- Ns Aq Cm delete
+Rrmove all characters before the cursor.
+.It Cm C-k
+Remove all characters after the cursor.
+.It Cm C-w
+Clear the filter.
+.It Cm M- Ns Op 0\(en9
+Print selected items and exit with a custom error code.
+See
+.Sx EXIT STATUS .
+.El
+.Sh ENVIRONMENT
+.Bl -tag -width "BEMENU_RENDERERS "
+.It Ev BEMENU_OPTS
+An alternative way to pass command line arguments to
+.Nm .
+.It Ev BEMENU_BACKEND
+Force a backend to be used.
+If empty, one of the GUI backends (Wayland, X11) will be selected
+automatically.
+The accepted values are
+.Cm curses , wayland
+and
+.Cm x11 .
+.It Ev BEMENU_RENDERER
+Force a backend by loading its shared object from the set value.
+.It Ev BEMENU_RENDERERS
+Override the backend search path to the set value.
+Defaults to
+.Pa @LIBDIR@/bemenu .
+.It Ev BEMENU_SCALE
+Override the rendering scale factor for the GUI backends.
+.El
+.Sh EXIT STATUS
+.Nm
+exits 0 on success, 1 if the user exited without selecting or an error,
+and if exiting with a custom error code, 10 plus the number specified.
+.Sh SEE ALSO
+.Xr dmenu 1 ,
+.Xr fork 2 ,
+.Xr ncurses 3 ,
+.Xr tty 4
+.Pp
+.Rs
+.%T Pango Reference Manual
+.%D August 20, 2020
+.%U https://developer.gnome.org/pango/1.46/
+.%O Fonts \(em pango_font_description_from_string()
+.Re
+.Sh AUTHORS
+.Nm
+is written and maintained by
+.An Jari Vetoniemi Aq Mt mailroxas@gmail.com .
```

Comment:
Feel free to add any other major contributors.

Reply:
Might be better as separate contributors file. The current list of contributors can be seen with `git shortlog -s -n --all --no-merges`. Anyhow, don't have to include that in scope of this PR.